Library/Homebrew/utils/json.rb

```diff
@@ -24,7 +24,7 @@ def stringify_keys(obj)
         obj.inject({}) do |result, (key, val)|
           key = key.respond_to?(:to_s) ? key.to_s : key
           val = stringify_keys(val)
-          result.merge!(key => val)
+          result[key] = val
```

Comment:
That's a tricky one. The CI errors are because the assignment and thus the block is not returning the modified hash `result` (like `Hash#merge!` did). You'll have to add a line with just `result` after this line.